src/components/exercises/00002-rr-no1/02-molecules/TableHeader/TableHeader.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './TableHeader.scss';
+
+const TableHeader = () => {
+    return (
+        <thead className={styles.wrapper}>
+            <tr className={styles.row}>
+                <th className={styles.item}>Make</th>
```

Comment:
There is no need to assign a class to every `th` element. You can style these elements easily with a proper [CSS selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).